txauthor.go

```diff
@@ -32,6 +32,10 @@ func (mw *MultiWallet) NewUnsignedTx(sourceWallet *Wallet, sourceAccountNumber i
 }
 
 func (tx *TxAuthor) AddSendDestination(address string, atomAmount int64, sendMax bool) {
```

Comment:
Address [this](https://github.com/raedahgroup/dcrlibwallet/pull/117#discussion_r395984445) review and run `go mod tidy`.